scripts/i18n-helper.js

```diff
@@ -0,0 +1,72 @@
+export default class I18nHelper {
+  constructor() {}
+
+  /**
+   * If midi-qol module is present and active
+   *
+   * @readonly
+   * @memberof I18nHelper
+   */
+  get _usesMidiQol() {
+    return game.modules.get('midi-qol')?.active;
+  }
+
+  i18nFormatFromKey(key, data = {}) {
+    return this._prepareI18n(key, data);
+  }
+
+  i18nFromKey(key) {
+    return this._prepareI18n(key, null);
+  }
+
+  i18nFromEffect(effect) {
+    return this._prepareI18n(effect.customId, null);
+  }
+
+  i18nFormatFromEffect(effect, data = {}) {
+    return this._prepareI18n(effect.customId, data);
+  }
+
+  _prepareI18n(customIdOrKey, data) {
+    // TODO Check if is a customId effect or a standard key i18n
+
+    let label = '';
+
+    // Manage a "try and get" on DND5e and if nothing is founded use the module setting...
+    // this can be useful in some case or for some strange update of DnD5e system...
+
+    // This is a example done for the conditions ... if you like it we can do the same thing
+    // for the feature and the spell SRD
+
+    // An example that comes to mind is that the Dnd5e system has the label for "Exhaustion",
```

Comment:
Maybe we just don't use DND5e's keys for exhaustion? I'm not sure how much that would simplify this

Reply:
It was just an example, but in general is the mechanism "find and replace" for Dnd5e labels okay for you as set? Personally if it is only for the conditions I think it a little overkill, but just tell me how did you prefer it and I adapt.